src/ynEIGEN/TokenStakingNode.sol

```diff
@@ -123,76 +175,92 @@ contract TokenStakingNode is ITokenStakingNode, Initializable, ReentrancyGuardUp
     //-------------------------------- EIGENLAYER WITHDRAWALS  -----------------------------
     //--------------------------------------------------------------------------------------
 
+    /**
+     * @notice Returns the queued shares and withdrawn balance for a specific strategy and asset.
+     * @dev The queued shares are the sum of the legacy queued shares and the post slashing queued shares.
+     */
     function getQueuedSharesAndWithdrawn(IStrategy _strategy, IERC20 _asset) external view returns (uint256, uint256) {
-        return (queuedShares[_strategy], withdrawn[_asset]);
+        return (preELIP002QueuedSharesAmount[_strategy] + queuedShares[_strategy], withdrawn[_asset]);
     }
 
     /**
      * @notice Queues withdrawals for a specific strategy
      * @param _strategy The strategy from which to withdraw
-     * @param _shares The number of shares to withdraw
+     * @param _depositShares The number of deposit shares to withdraw. The amount of withdrawn shares on completion might be lower due to slashing.
      * @return _fullWithdrawalRoots An array of withdrawal roots generated by the queueWithdrawals operation
      */
-    function queueWithdrawals(IStrategy _strategy, uint256 _shares)
+    function queueWithdrawals(IStrategy _strategy, uint256 _depositShares)
         external
         onlyTokenStakingNodesWithdrawer
         onlyWhenSynchronized
         returns (bytes32[] memory _fullWithdrawalRoots)
     {
         IStrategy[] memory _strategiesArray = new IStrategy[](1);
         _strategiesArray[0] = _strategy;
-        uint256[] memory _sharesArray = new uint256[](1);
-        _sharesArray[0] = _shares;
-        IDelegationManager.QueuedWithdrawalParams[] memory _params = new IDelegationManager.QueuedWithdrawalParams[](1);
-        _params[0] = IDelegationManager.QueuedWithdrawalParams({
+        uint256[] memory _depositSharesArray = new uint256[](1);
+        _depositSharesArray[0] = _depositShares;
+        IDelegationManagerTypes.QueuedWithdrawalParams[] memory _params = new IDelegationManagerTypes.QueuedWithdrawalParams[](1);
+        _params[0] = IDelegationManagerTypes.QueuedWithdrawalParams({
             strategies: _strategiesArray,
-            shares: _sharesArray,
-            withdrawer: address(this)
+            depositShares: _depositSharesArray,
```

Comment:
`_depositsharesArray` is called `shares` in the other PR

Reply:
actually there are other naming differences, you should check the same function in the other PR and align the names. I prefer the names in the other PR.